README.adoc

```diff
@@ -233,9 +233,9 @@ include::{common-includes}/mvnpackage.adoc[]
 
 == Testing the application
 
-While you can test your application manually, you should rely on automated tests since they will
-trigger a failure whenever a code change introduces a defect. Since the application is a RESTful 
-web service application, you can use JUnit and the RESTful web service Client API to write tests. 
+Although you can test your application manually, automated tests have the advantage of
+triggering a failure whenever a code change introduces a defect. Because the application is a RESTful 
+web service application, you can use JUnit and the RESTful Web Service Client API to write tests. 
```

Comment:
After some research and some thought, I think "RESTful Web Service Client API" can be "REST Client API".